notebooks/official/custom/custom_training_tensorboard_profiler.ipynb

```diff
@@ -61,7 +61,7 @@
       "source": [
```

Comment:
<p>Simplify this cell to:</p><p> </p><p><code>TENSORBOARD_NAME = "your-tensorboard-unique"&nbsp;# @param {type:"string"}</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1266/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>